### PR TITLE
refactor screens

### PR DESCRIPTION
**Pull request by caudaganesh**
- [x] Create screens indexes
- [x] Add helpers
- [x] Refactor local config - rename "local.example.js" to "local.js" and fill the SOUNDCLOUD_CLIENT_ID value.
- [x] Separate style

### CODE REVIEW

File: src/screens/PessageScreen/styles.js [hunk 1]
```diff
@@ -0,0 +1,188 @@
+import { StyleSheet, Platform } from 'react-native';
```
Comment (sonnylazuardi) [inline src/screens/PessageScreen/styles.js]: @caudaganesh I think no need to separate styles, just try to separate components. Components and style should be in one file i think

File: src/helpers.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+export const bookNameHelper = (value) => {
```
Comment (sonnylazuardi) [inline src/helpers.js]: I like this helper structures 👍

File: src/screens/PessageScreen/styles.js (hunk 1 above)
Comment (caudaganesh) [reply src/screens/PessageScreen/styles.js]: @sonnylazuardi
okay i will put it back together, 
seems like someone already work on that issue on refactoring into smaller components?

File: src/helpers.js (hunk 2 above)
Comment (caudaganesh) [reply src/helpers.js]: thanks, i think we need more modular structure for this project.

File: src/screens/PessageScreen/index.js [hunk 3]
```diff
@@ -20,14 +20,15 @@ import {
 } from "react-native";
 const Realm = require("realm");
 import DrawerLayout from "react-native-drawer-layout";
-import Books from "../constants/Books";
+import Books from "../../constants/Books";
```
Comment (sonnylazuardi) [inline src/screens/PessageScreen/index.js]: why do we need to rename `src/screens/PassageScreen.js → src/screens/PessageScreen/index.js`? I think it's better to keep the filename (I followed the convention in the expo project to keep the filename)

Comment (caudaganesh) [reply src/screens/PessageScreen/index.js]: Okay, I will rename it. Are we gonna consider using redux in this project?

Comment (sonnylazuardi) [reply src/screens/PessageScreen/index.js]: yea i think for features like history we can use redux and save it to realm

File: src/app.js [hunk 4]
```diff
@@ -1,8 +1,8 @@
 //@flow
 import React, { Component } from "react";
 import { StyleSheet, View, Platform, StatusBar, Modal } from "react-native";
-import PassageScreen from "./screens/PassageScreen";
-import SearchScreen from "./screens/SearchScreen";
+import PassageScreen from "./screens/PessageScreen/PessageScreen";
```
Comment (sonnylazuardi) [inline src/app.js]: typo should be Passage. I think it's better to create folder like this `./screens/Passage/PassageScreen`

Comment (caudaganesh) [reply src/app.js]: omg. sorry, okay work on it

**Pull request merged**